source/FreeRTOS_DNS.c

```diff
@@ -207,7 +207,7 @@
         static uint32_t prvPrepareLookup( const char * pcHostName )
     #endif
     {
-        uint32_t ulIPAddress = 0UL;
+        uint32_t ulIPAddress = ( uint32_t ) 0UL;
```

Comment:
@alfred2g, as I commented in another PR about MISRA: please do not use the "L" suffix, just write:
~~~c
 uint32_t ulIPAddress = 0U;
~~~